testing/tests/DevExpress.core/utils.format.tests.js

```diff
@@ -135,3 +137,17 @@ QUnit.test("percent format parsing", function(assert) {
     assert.strictEqual(parser("-10.0%"), -0.1, "parse negative float number with 1 digit");
     assert.strictEqual(parser("-10%"), null, "negative value without float part should be incorrect");
 });
+
+
+QUnit.module("number formatter");
+
+QUnit.test("float with precision formatting", function(assert) {
+    var formatter = generateNumberFormat("#.00");
+
+    assert.strictEqual(formatter(null), "0.00", "format an empty value");
+    assert.strictEqual(formatter(0), "0.00", "format zero");
+    assert.strictEqual(formatter(123), "123.00", "format integer");
+    assert.strictEqual(formatter(123.5), "123.50", "format rounded float");
+    assert.strictEqual(formatter(123.57), "123.57", "format float");
+    assert.strictEqual(formatter(-123.57), "-123.57", "format negative float");
```

Comment:
Add test case with 3 numbers after float point.  For example, 123.577. The last digit should be rounded.